tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,58 @@
+"""Work with inventory."""
+
+
+def display_inventory(inventory):
+    """Print inventory."""
+    print('Inventory:')
+    items_count = 0
+    # check that inventory is a dict
+    if not isinstance(inventory, dict):
+        print('Wrong inventory type. Should be dict')
+        return
+    # print inventory and calculate sum of items
+    for key, value in inventory.items():
+        print(value, key)
+        items_count += value
+    print()
+    # check number of item and print according caution if needed
+    print('Total number of items:', items_count, end='\n\n')
+    if 60 <= items_count <= 69:
+        print('CAUTION: Your backpack weighs a lot, your stamina runs out quicker!')
+    elif 70 <= items_count <= 79:
```

Comment:
I think it would be good to introduce constants here.

eg.
MAX_INVENTORY_CAP = 80
HEAVY_WEIGHT_THRESHOLD = 70
LIGHT_WEIGHT_THRESHOLD=60

Feel free to use other naming names if u come up with better ones :D
Also, please double-check the numbers I've provided here, that's just a proposition.

And of course, consts should be moved to the top of the file.